examples/light-switch-app/silabs/README.md

```diff
@@ -254,23 +254,9 @@ combination with JLinkRTTClient as follows:
           <info  > [SVR] Copy/paste the below URL in a browser to see the QR Code:
           <info  > [SVR] https://project-chip.github.io/connectedhomeip/qrcode.html?data=CH%3AI34NM%20-00%200C9SS0
 
-    **LED 0** shows the overall state of the device and its connectivity. The
-    following states are possible:
-
-        -   Short Flash On (50 ms on/950 ms off): The device is in the
-            unprovisioned (unpaired) state and is waiting for a commissioning
-            application to connect.
-
-        -   Rapid Even Flashing (100 ms on/100 ms off): The device is in the
-            unprovisioned state and a commissioning application is connected through
-            Bluetooth LE.
-
-        -   Short Flash Off (950ms on/50ms off): The device is fully
-            provisioned, but does not yet have full Thread network or service
-            connectivity.
-
-        -   Solid On: The device is fully provisioned and has full Thread
-            network and service connectivity.
+    **LED 0** shows the overall state of the device and its connectivity.
+    However, this status LED is disabled by default on Silabs ICD Light-switch
+    application.
```

Comment:
That isn't entirely true. With ICD on, it will only activate for the factory reset sequence or an Identify command (identify cluster).

If for some reason someone built this app disabling  ICD configs off, he led would behave regularly.

Reply:
Updated to include behavior for both configurations